src/site/content/en/blog/reporting-api-migration/index.md

```diff
@@ -0,0 +1,471 @@
+---
+layout: post
+title: Migrate to the Reporting API v1
+subhead: |
+  A new version of the Reporting API is available. It's simpler, more private, and more likely to be supported across browsers.
+authors:
+  - maudn
+date: 2021-06-14
+updated: 2021-06-14
+description: |
+  Migrate to the new Reporting API.
+hero: image/O2RNUyVSLubjvENAT3e7JSdqSOx1/PYEe5UP3bVYzPMXdQc0X.jpg
+alt: |
+  A person and their refection on wet sand, symbolizing the migration from the legacy to the new Reporting API.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - monitoring
+  - security
+---
+
+{% Banner 'caution', 'body' %} This post covers how to migrate to the new version of the
+Reporting API (v1). If you're new to the Reporting API or need detailed usage examples,
+head over to the [main post on the Reporting API](/reporting-api) instead. {% endBanner %}
+
+The Reporting API informs you about errors that happen across your site as visitors use
+it. It gives you visibility on browser interventions, browser crashes,
+Content-Security-Policy violations, COOP/COEP violations, deprecation warnings and more.
+
+**A new version of the Reporting API is available. It's simpler, more private, and more
+likely to be supported across browsers.**
+
+{% Aside %} The legacy Reporting API is named _Reporting API v0_. The new Reporting API is
+named _Reporting API v1_. {% endAside %}
+
+## Summary
+
+{% Banner 'info', 'body' %}
+
+If you already have reporting functionality for your site, **migrate to v1**
+(`Reporting-Endpoints` header) but keep the v0 header (`Report-To`) around for some time.
+
+If you're adding reporting functionality to your site just now, use only v1 (`Reporting-Endpoints` header).
+
+Make sure to set the `Reporting-Endpoints` header on all responses that might generate
+reports.
+
+If you're maintaining an endpoint service or are operating your own, expect **more
+traffic** as you or external developers migrate to v1 (`Reporting-Endpoints` header).
+
+{% endBanner %}
```

Comment:
```suggestion

If you already have reporting functionality for your site, **migrate to v1**
(`Reporting-Endpoints` header) but keep the v0 header (`Report-To`) around for some time.

If you're adding reporting functionality to your site just now, use only v1 (`Reporting-Endpoints` header).

Make sure to set the `Reporting-Endpoints` header on all responses that might generate
reports.

If you're maintaining an endpoint service or are operating your own, expect **more
traffic** as you or external developers migrate to v1 (`Reporting-Endpoints` header).

```

Reply:
This is too much text for a banner, I think it's easier to read on a plain background. It's already in its own section that should be enough.